File: Lua/4DaysORM/src/Models/Department.lua
local Table = require "orm.model"
local fields = require "orm.tools.fields"

local Department = Table{
    __tablename__ = "departments",
    name = fields.CharField{ max_length = 50, unique = false },
}

Department.to_string = function (dep)
    return "Department { Id: " .. dep.id .. ", Name: " .. dep.name .. " }"
end

return Department

File: Python/Peewee/models/department.py
from models.base_model import BaseModel

from peewee import AutoField, CharField


class Department(BaseModel):
    id: int = AutoField()
    name: str = CharField()

    class Meta:
        table_name = 'departments'
        model_name = 'Department'

    def __str__(self):
        return f'{{ Department: {{ Id: {self.id}, Name: {self.name} }} }}'
